packages/server/__tests__/inviteToTeam.test.ts

```diff
@@ -1,71 +0,0 @@
-import {getUserTeams, sendPublic, signUp} from './common'
-
-test('Invite to team, works for ordinary email domain', async () => {
-  const {userId, authToken} = await signUp()
-  const {email: inviteeEmail} = await signUp()
-
-  const teamId = (await getUserTeams(userId))[0].id
-  const inviteToTeam = await sendPublic({
-    query: `
-      mutation InviteToTeam($teamId: ID!, $invitees: [Email!]!) {
-        inviteToTeam(teamId: $teamId, invitees: $invitees) {
-          error {
-            title
-            message
-          }
-          invitees
-        }
-      }
-    `,
-    variables: {
-      teamId,
-      invitees: ['foo@example.com', inviteeEmail]
-    },
-    authToken
-  })
-
-  expect(inviteToTeam).toMatchObject({
-    data: {
-      inviteToTeam: {
-        error: null,
-        invitees: ['foo@example.com', inviteeEmail]
```

Comment:
We need to make sure the email addresses are valid now, but the `inviteeEmail` from `signUp` is invalid. 

If we include a real email address, we'll keep pinging that email when running the test. We could create a valid Parabol email just for this? E.g. tests@parabol.co

Reply:
-1 We're not testing with `MailManagerMailgun`, do we?

Reply:
Could you elaborate, please? 

The test is testing `inviteToTeam`, which is using `inviteToTeamHelper`. In the test, we have the `signUp` helper function with provides a random email address.

Previously, our custom logic would send emails to any email address (excluding tempmail.cn and qq.com domains). 

Now, that we're using Mailgun's validate logic in `inviteToTeamHelper`, we won't send emails to the randomly generated ones if they're undeliverable. 

Here's an example response where the test fails:

```
      Object {
        "data": Object {
          "inviteToTeam": Object {
    -       "error": null,
    -       "invitees": Array [
    -         "foo@example.com",
    -         "eusebio_koelpin82@yahoo.com",
    -       ],
    +       "error": Object {
    +         "message": "No valid emails",
    +         "title": null,
    +       },
    +       "invitees": null,
          },
        },
      }
```